api/definition/swagger.yaml

```diff
@@ -0,0 +1,273 @@
+openapi: 3.0.0
+servers:
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/ANDREWJEVANS/fingertips-next-api/1.0.0
+info:
+  description: An API to query public health indicator data.
+  version: "1.0.0"
+  title: Fingertips API
+  contact:
+    email: you@your-company.com
```

Comment:
We probably want to either leave this out or add the actual Fingertips feedback email.